feature/githubRepoSearch/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-transform-react-jsx",
+      "@babel/plugin-transform-runtime"
+    ]
+  }
+  
```

Comment:
Please add an empty line at the end of each file.

Reply:
Done. I will keep this in mind for the future.